### ISSUE TITLE
missleading "redoCompleted" label

### ISSUE THREAD

**Issue by i9hdkill** [FlyingPikachu/Quests#126]
Hi,
I like the new "Completed" label in our quest Guis, however it's quite missleading.

We do have daily and weekly quests available from different npcs. Players check these nps daily to get the daily/weekly quest again.

So far so good. For the first time they click and accept the quest and complete it.
On the following day, this quest is marked as "Completed" in gui. This is technically right, as they did the quest yesterday.
But it's counter-intuitive, as they now think, they can't do the quest again.(But they can, as it's a daily one) You know what I am trying to say?

Two possible solutions:
Only show Completed Label on one-time-quests

or

Or only show the completed label on time-gated quests when the cooldown is active + Show a line with:
Cooldown (translatable) +
remaining time (Don't need to be live updated)
In other words:
Quest is on cooldown: Show Completed Label + new line with Cooldown : 23h 2m
Quest is available again: Remove Completed Label and Cooldown line

I like the second solution

**Comment by FlyingPikachu** [comment]
Duplicate of #93 

Although the options you listed were considered, I came to the conclusion that the mere fact the quest appears in the GUI should indicate that it is available. The word "Completed" is intended to show it's been completed _at least once_ but I understand your point, which is why the word can be changed or removed through the language file.

Feel free to continue this discussion in #93

**Comment by i9hdkill** [comment]
I was aware of #93 , but 93 is a waaay more advanced and harder to implement. My changes are (as far as I know) doable with relatively minor code changes with still a big improvement compared to the actual state of the plugin.
Whats the gain for the player, if they know that they did a repeatable quest once? Completed means it's done. Not doable and already done for this time period.

**Comment by FlyingPikachu** [comment]
I'm not sure what you mean by "more advanced," but the majority of #93 has already been implemented. Please continue this discussion there.
